### PR TITLE
Add missing support for bucket sort order in Glue

### PR DESCRIPTION
**Pull request by sopel39**

### CODE REVIEW

**Comment by sopel39** [comment]
We should add Glue tests for these too

**Comment by findepi** [comment]
> We should add Glue tests for these too

there is `TestHiveGlueMetastore`
maybe it can be adapted for these needs.

File: presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/converter/GlueInputConverter.java [hunk 1]
```diff
@@ -99,9 +103,15 @@ private static StorageDescriptor convertStorage(Storage storage, List<Column> co
         sd.setOutputFormat(storage.getStorageFormat().getOutputFormatNullable());
         sd.setParameters(ImmutableMap.of());
 
+        Optional<HiveBucketProperty> bucketProperty = storage.getBucketProperty();
```
Comment (findepi) [inline presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/converter/GlueInputConverter.java]: use this var in the next line

**Comment by findepi** [comment]
AFAICT we respect bucket sort order when _reading_ glue table, but worth double checking.

**Comment by sopel39** [comment]
> AFAICT we respect bucket sort order when reading glue table, but worth double checking.

Yes, we do: `io.prestosql.plugin.hive.metastore.glue.converter.GlueToPrestoConverter#setStorageBuilder`

**Pull request merged**